workspaces/global-header/plugins/global-header/src/components/HeaderDropdownComponent/ProfileDropdown.tsx

```diff
@@ -82,7 +83,15 @@ export const ProfileDropdown = ({ layout }: ProfileDropdownProps) => {
         <Box sx={{ display: 'flex', alignItems: 'center', ...layout }}>
           {displayName && (
             <>
-              <AccountCircleOutlinedIcon fontSize="small" sx={{ mr: 1 }} />
+              {profile.picture ? (
+                <Avatar
+                  src={profile.picture}
+                  sx={{ mr: 1, height: '32px', width: '32px' }}
```

Comment:
A tiny change request but it looks better with more margin since the profile picture now is lager than the small icon.
```suggestion
                  sx={{ mr: 2, height: '32px', width: '32px' }}
```
mr: 2
![image](https://github.com/user-attachments/assets/91727730-e12a-4d36-8d1e-b4b8011d3bd3)
mr: 1
![image](https://github.com/user-attachments/assets/e5d5359a-b49c-4391-bf88-95220fd3d60f)